csds-client/client/v2/client.go

```diff
@@ -0,0 +1,290 @@
+package clientV2
```

Comment:
Up to you, but you don't have to call it v2, as v2 is implied by the package path. See "package path" in https://blog.golang.org/package-names

Reply:
+1

Reply:
I've just renamed it to `client`